### ISSUE TITLE
CrossEntropy --> NLLLoss?

### ISSUE THREAD

**Issue by lezasantaizi** [XenderLiu/Listen-Attend-and-Spell-Pytorch#3]
logits is logsoftmax

**Comment by kevin1kevin1k** [comment]
Thanks for the comment.
Actually `CrossEntropyLoss` equals `LogSoftmax` combined with `NLLLoss`, so we think using CrossEntropyLoss is fine.
If I misunderstand your point, pleaes kindly let us know.

Thanks.
